### PR TITLE
Documentation for TestContainers

### PR DESCRIPTION
**Pull request by DoodleBobBuffPants**

### CODE REVIEW

File: documentation/database/testcontainers.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Driver
+<table class="table">
+<tr>
+<th>URL format</th>
+<td><code>jdbc:tc:</code> instead of <code>jdbc:</code> for your database</td>
+</tr>
+<tr>
+<th>Ships with Flyway Command-line</th>
+<td>Yes</td>
```
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: afaik we arent shipping this with the cli

File: documentation/database/testcontainers.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+</tr>
+<tr>
+<th>Maven Central coordinates</th>
+<td><code>org.testcontainers:jdbc:jar:1.14.3</code></td>
```
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: we should probably list all the jars required, so the final ones (postgres, mysql, etc) and all the other dependencies

File: documentation/database/testcontainers.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### Example URL
+
+```
+jdbc:tc:postgresql:11-alpine://localhost:5432/databasename
+```
+
+## Limitations
+
+- If Flyway doesn't ship with the JAR for your chosen database, you will still need to provide it in order to use it with TestContainers
```
Comment (Lyeeedar) [inline documentation/database/testcontainers.md]: Seeing as we dont ship it, we can remove the 'if' part

**Review by Lyeeedar — CHANGES_REQUESTED** [review]

File: documentation/database/testcontainers.md (hunk 3 above)
Comment (DoodleBobBuffPants) [reply documentation/database/testcontainers.md]: It's the TestContainers JAR we don't ship, but this refers to the actual database driver. TestContainers still requires one of those to be present.

Comment (Lyeeedar) [reply documentation/database/testcontainers.md]: That should be more explicit then, maybe with an example (i.e. 'if using the postgres test container $postgresdependency$, you also need to include etc etc')

**Review by Lyeeedar — APPROVED** [review]

**Pull request merged**